src/libsystemd/sd-path/sd-path.c

```diff
@@ -321,23 +321,23 @@ static int get_path(uint64_t type, char **buffer, const char **ret) {
                 return from_user_dir("XDG_DESKTOP_DIR", buffer, ret);
 
         case SD_PATH_SYSTEMD_UTIL_DIR:
-                *ret = ROOTPREFIX "lib/systemd";
+                *ret = ROOTPREFIX "/lib/systemd";
```

Comment:
Indeed, this shows (before your patch):
```console
$ build/systemd-path systemd-util-dir
/usrlib/systemd
```
I looked at the output many times when writing this, but somehow I didn't see this one.